test/unit/lib/redmine/i18n_test.rb

```diff
@@ -83,6 +83,13 @@ def test_time_format
         assert_equal '2011-02-20 15:45', format_time(now)
         assert_equal '15:45', format_time(now, false)
       end
+   end
```

Comment:
`end` at 86, 3 is not aligned with `with_settings :time_format => '%H:%M' do` at 76, 4